scripts/us_census/acs5yr/subject_tables/common/base.py

```diff
@@ -0,0 +1,238 @@
+"""Base data loader for ACS subject tables"""
```

Comment:
It's a bit hard to follow what's going on.

We spoke about having two scripts. First one that takes the config and needs to read only the CSV header, and produces the column->{stat-var, unit, etc.}.  The second one, that takes the config again (for denominator) and output from the first and goes over every csv. If that's the plan, why does this script load and process the CSV?

It might also be useful to define the interface first.  A few things need to happen, and its better to them to be modular:
1. Downloading stuff
2. Generating SVs from spec
3. Generating cleaned CSVs from spec + output of #2.

Finally, it might also be useful to have a simple config checker that ensures that every column/column-component referred to in the spec appears in the header. To guard against any typos.

Reply:
Thank you for the review. I have modularized the code. In this PR, the focus will be implementation of (2) Generating SVs from spec, with a unit test for the same. I will make another PR, where we add (3)

Reply:
Puzzled why base.py is in this PR. Can we just include generate_column_map?

Reply:
Skipping this for further review in this PR.